src/systemctl/systemctl.c

```diff
@@ -4767,7 +4767,7 @@ static int show_one(
         else if (streq(verb, "status")) {
                 print_status_info(bus, &info, ellipsized);
 
-                if (info.active_state && STR_IN_SET(info.active_state, "inactive", "failed"))
+                if (info.active_state && STR_IN_SET(info.active_state, "inactive", "failed", "activating"))
```

Comment:
"deactivating" should not be considered running either. I figure we should invert the check here, and use `!STR_IN_SET(info.active_state, "active", "reloading")`